grpc/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "grpc"
+version = "0.9.0"
+edition = "2021"
+authors = ["gRPC Authors"]
+license = "Apache-2.0"
+
+[dependencies]
+url = "2.5.0"
+tokio = { version = "1.37.0", features = ["sync", "full" ] }
```

Comment:
we should reduce the feature flags here, if we need a runtime add it as a dev-dep

Reply:
Is `sync` right here, then?

Reply:
Yeah, if the library item you need is within the sync crate then you can just include that feature. There should be a feature flag overview in the main docs of tokio.